### ISSUE TITLE
google-analytics-chart ignores "width" attribute.

### ISSUE THREAD

**Issue by rchedrese-va** [GoogleWebComponents/google-analytics#19]
The "width" attribute of the google-analytics-chart component doesn't appear to do anything. It just passes the value to the "width" attribute of google-chart which again does nothing. From what I can tell the proper way to set the width of a chart is through the "width" style attribute. 

This works:
```html
<google-chart style="width: 100px"></google-chart>
```

This does not:
```html
<google-chart width="100"></google-chart>
```

**Comment by philipwalton** [comment]
Thanks for reporting this. It looks like `<google-chart>` [switched to preferring CSS for width/height styling](https://github.com/GoogleWebComponents/google-chart/pull/17/files) in a recent release. I'll update the documentation.

**Comment by rchedrese-va** [comment]
No problem. The ```<google-analytics-chart>``` component will also probably need to be updated to set the css width/height for the ```<google-chart>```.

**Comment by bamnet** [comment]
+1  I ran into the same issue and ended up using some css like 
```css
google-analytics-chart > ::content google-chart {
  width: 600px;
}
```

**Comment by philipwalton** [comment]
@bamnet thanks for reminding me about this issue. Your fix shouldn't be needed after #33 is merged in.
